md/operations.md

```diff
@@ -31,6 +31,8 @@ You can define a sequence of operations at a task. The operations are performed
    Go to [Initialize BDM attributes from contract inputs in task operations](define-and-deploy-the-bdm.md) for more information.  
 * _**isDeleted**_: an operator 'Is Deleted' can be called on a business data variable. The BO instance that the business data references is deleted from the database, at runtime, after the task is completed. The process designer should avoid calling other operations on the same business data variable within the same group of operations/actions.
```

Comment:
So you should remove the last part of this sentence, as you already detail it below